### PR TITLE
Update ITEM.tsv

### PR DESCRIPTION
**Pull request by Sunaries**
- Fixed a portion of the many instances from IMC's pull request (https://github.com/Treeofsavior/EnglishTranslation/commit/2ebd07ddaf15c1d4ea174179cf623018ecfec5bb) that could be improved.

### CODE REVIEW

File: ITEM.tsv [hunk 1]
```diff
@@ -5567,11 +5567,11 @@ ITEM_20150717_005566	The first time a device was attached to the tip of a bow to
 ITEM_20150717_005567	This bow requires a lot more strength from the user than expected. The view of the arrow in flight will leave a sense of satisfaction.
 ITEM_20150717_005568	Unlike its name, it will be welcomed by any archer. Only rogues dislike it.
 ITEM_20150717_005569	The burden of bows has been alleviated, it does not decrease the power by much.
-ITEM_20150717_005570	- All Hunter Skill Levels increased by +2
+ITEM_20150717_005570	- Increased all Hunter skill levels by 2
 ITEM_20150717_005571	- Basic Attacks have 2% chance to poison the target for 20 seconds
-ITEM_20150717_005572	If you don't have the other bow, then this bow is good choice.
-ITEM_20150717_005573	Weapon Merchant said that even Lydia Scaffen seems to be buying it. It is not wrong saying since she is never complain the bow.
-ITEM_20150717_005574	- 100% Critical chance to Beast types{nl} - 25% decreased damage to non-Beast types
+ITEM_20150717_005572	If you don't have the other bow, then this bow is good choice to buy.
```
Comment (ghost) [inline ITEM.tsv]: "then this bow is good choice to buy." -> "then this bow is a good choice to buy."

File: ITEM.tsv [hunk 2]
```diff
@@ -5889,114 +5889,114 @@ ITEM_20150729_005885	Weeds
...
-ITEM_20150804_005992	A title for who is the first player reached 6 Rank during 1 CBT. Right-click to use.
-ITEM_20150804_005993	Title - Ausurine Hero
-ITEM_20150804_005994	A title for who is the player between 2~10 reached 6 Rank during 1 CBT. Right-click to use.
-ITEM_20150804_005995	The First Ordeal Symbol of the Great King
-ITEM_20150804_005996	A symbol that means to pass the test prepared by Zachariel. It seems to related to Royal Tomb's seal.
-ITEM_20150804_005997	The Second Ordeal Symbol of the Great King
-ITEM_20150804_005998	The Third Ordeal Symbol of the Great King
-ITEM_20150804_005999	The Fourth Ordeal Symbolf the Great King
+ITEM_20150804_005991	Title - Ausrine's Incarnation
+ITEM_20150804_005992	A title given to the first player who have reached the 6th rank during the 3rd CBT. Right-click to use.
```
Comment (ghost) [inline ITEM.tsv]: have -> has

**Comment by Sunaries** [comment]
:+1:  Good to merge. 

Note: I have trouble coming up with a better item name for "과거의 기억을 기록하는 보주" _(ITEM_20150729_005909)_ - What you entered isn't 100% correct. It should be something along the lines of "Recording/Writing Past Memories Orb" ?

**Comment by ghost** [comment]
Perhaps "Memory Recording Orb"?

**Comment by imcgames** [comment]
Thank you :smile: 

(so... will it be changed to Memory Recording Orb? )

**Pull request merged**